src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,47 @@
+import Button from "../../components/Button/Button";
+import "./Profile.scss";
+import { Link } from "react-router-dom";
+import { UserProfile } from "./mockData";
+
+type ProfileProps = {
+  user: UserProfile;
+};
+
+const Profile = ({ user }: ProfileProps) => {
```

Comment:
Can you find a way to use deconstruction here?

Reply:
You're cheeky you are